src/faction_camp.cpp

```diff
@@ -1875,8 +1875,12 @@ void basecamp::job_assignment_ui()
                     if( smenu.ret == UILIST_CANCEL ) {
                         break;
                     }
-                    if( smenu.ret == 0 ) {
-                        cur_npc->job.clear_all_priorities();
+                    if( smenu.ret == 0 )
+                    {
+                        if( query_yn( _( "Are you sure you wish to clear priorities for %s?"),cur_npc->disp_name() ) )
+                        {
+                            cur_npc->job.clear_all_priorities();
+                        }
```

Comment:
```suggestion
                    if( smenu.ret == 0 ) {
                        if( query_yn( _( "Are you sure you wish to clear priorities for %s?" ), cur_npc->disp_name() ) ) {
                            cur_npc->job.clear_all_priorities();
                        }
```

Reply:
Urgh, I need to do some configuring. I assumed the CB project file would include settings for the source formatter, but it appears not. In retrospect I'm not even aware if it can. Thanks!